filebeat/prospector/redis/harvester.go

```diff
@@ -131,9 +131,7 @@ func (h *Harvester) Run() error {
 			"redis": common.MapStr{
 				"slowlog": subEvent,
 			},
-			"beat": common.MapStr{
-				"read_timestamp": common.Time(time.Now()),
-			},
+			"read_timestamp": common.Time(time.Now()),
```

Comment:
Yeah, now I remember we even had a quick discussion. +1 on changing it but not sure what to :-)